examples/datadog-express-apm/container-agent/docker-compose.yml

```diff
@@ -1,13 +1,20 @@
-version: '3.5'
 services:
   app:
-    build: .
+    build:
+      context: .
+    env_file:
+      - ./src/.env
     environment:
+      - DD_SITE=us3.datadoghq.com
+      - DD_API_KEY=7ff7e8c4d8baac789735d8cf0f159987
+      - NODE_ENV=production
```

Comment:
Won't it be nice read these values from environment during build time?

Example:
```
DD_SITE=${DD_URL}
DD_API_KEY=${DD_API_KEY}
...
```

Reply:
you are right, these are for test I am gonna remove them just forgot to do it